test/fileNameParser.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable semi, no-unused-expressions, no-undef */
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import sinonChai from 'sinon-chai'
+import { formatFileName } from '../src/helpers/fileNameParser'
+
+chai.should()
+chai.use(sinonChai)
+chai.use(chaiAsPromised)
+const { expect } = chai
+
+describe('helpers/fileNameParser', () => {
```

Comment:
Perhaps add a test rejecting a filename with no epub in it at all?

Reply:
Good idea, adding that test now.